python/py-pytest-xdist/Portfile

```diff
@@ -18,9 +18,9 @@ long_description    {*}${description}
 
 homepage            https://github.com/pytest-dev/pytest-xdist
 
-checksums           rmd160  b720976b2924ac7e660cb342e4b59545f6d44bb3 \
-                    sha256  718887296892f92683f6a51f25a3ae584993b06f7076ce1e1fd482e59a8220a2 \
-                    size    64956
+checksums           rmd160  9283fac66adbe218c9fba602b1c621b9d02e39bf \
+                    sha256  d5ee0520eb1b7bcca50a60a518ab7a7707992812c578198f8b44fdfac78e8c93 \
+                    size    77751
 
 python.versions     37 38 39 310 311
 
```

Comment:
please correct the dependencies:
- MacPort defaults to `python.pep517 yes` and the `setuptools` build backend for Python 3.7+ and thus some dependencies will be added automatically. You can remove the `setuptools` dependency fro `depends_lib`. 
- from the upstream `setup.cfg` file it appears that the only other dependencies are: `execnet` and `pytest`; please verify the others are indeed no longer needed and adapt the Portfile accordingly.

Reply:
Yes, https://github.com/pytest-dev/pytest-xdist/commit/1fbf49909ff9b2c33305295835caf1453fd4ae27 (remove pytest-forked dependency) and six no longer required (python 2.7 support dropped in pytest-xdist 2.0.0)